src/main/java/org/mifos/connector/gsma/identifier/IdentifierLookupRoutes.java

```diff
@@ -117,11 +131,20 @@ public void configure() throws Exception {
                 .toD(BaseURL + account + "/${exchangeProperty."+IDENTIFIER_TYPE+"}/${exchangeProperty."+IDENTIFIER+"}/${exchangeProperty."+ACCOUNT_ACTION+"}?bridgeEndpoint=true&throwExceptionOnFailure=false");
 
         /**
-         * Main route for account status
+         * Base route for accounts
+         * TODO: Add support for multiple identifier lookup
```

Comment:
It is to add support for a sub-identifier along with MSISDNs to identify an user. This will require some major changes, so I kept it as a TODO in the future.